presenters/trial.js

```diff
@@ -71,6 +71,7 @@ function decorateIdentifiers(identifiers, sources) {
 }
 
 function decorateTrial(trial) {
+  trial.records = _.sortBy(trial.records, 'source_id');  // eslint-disable-line no-param-reassign
```

Comment:
Instead of changing the records here, do it a few lines later,  like:

```javascript
    {
      documents: decorateDocuments(trial.documents || []),
      identifiers: decorateIdentifiers(trial.identifiers || {}, trial.sources || {}),
      records: // ...
    }
```

Check also if `trial.records` never is undefined. To be safe, it might be better to use `trial.records || []`.

Also, please add a test for this.